nest-desktop-launch

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+source /app/bin/nest_vars.sh
+
+# export NESTML_MODULES_PATH="${NESTML_MODULES_PATH:-/tmp/nestmlmodules}"
+# export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${NESTML_MODULES_PATH}
+
+export NEST_SERVER_DISABLE_AUTH=1
+export NEST_SERVER_DISABLE_RESTRICTION=1
+export NEST_SERVER_ENABLE_EXEC_CALL=1
+export NEST_SERVER_MODULES="import nest; import numpy; import numpy as np"
+
+# Start NEST Server as daemon
+export NEST_SERVER_DAEMON=1
+/app/bin/nest-server start
+
+# Start NESTML Server as daemon
+# export NESTML_SERVER_DAEMON=1
+# /app/bin/nestml-server start
+
+# Start NEST Desktop
+/app/bin/nest-desktop --no-sandbox
```

Comment:
use zypak from the electron baseapp, drop no-sandbox https://docs.flatpak.org/en/latest/electron.html#launching-the-app

Reply:
![image](https://github.com/user-attachments/assets/fdafab54-0faf-4153-8e1b-aba83adc3780)

Reply:
It already contains `zypack-wrapper.sh` in `/app/bin/nest-desktop`.

Reply:
then why are you passing no-sandbox?